### PR TITLE
home.php updated to get_template_part()

### PR DESCRIPTION
**Pull request by gravcmcguire**
Yo! How wack is this?

### CODE REVIEW

File: templates/home.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 $blog_page_id = get_option('page_for_posts');
 
-WPUtil\Component::render('components/banners/banner-default', [
+get_template_part('components/banners/', 'banner-default', [
```
Comment (jrcreative) [inline templates/home.php]: @gravcmcguire this should look more like this:

`get_template_part('components/banners/banner', 'default', [ ...args ]`

File: templates/home.php [hunk 2]
```diff
@@ -13,7 +13,7 @@
 	<?php
 	global $wp_query;
 
-	WPUtil\Component::render('components/posts-list/posts-list-blog', [
+	get_template_part('components/posts-list/posts-list-blog', [
```
Comment (jrcreative) [inline templates/home.php]: This needs a second argument for "name" before the args

`get_template_part('components/posts-list/posts-list', 'blog', [ ..args ]`

File: templates/home.php [hunk 3]
```diff
@@ -13,7 +13,7 @@
...
 		'wp_query_obj' => $wp_query
```
Comment (jrcreative) [inline templates/home.php]: Check to make sure `$wp_query` is a `WP_Query` object with `is_a($wp_query, 'WP_Query')`

**Review by jrcreative — CHANGES_REQUESTED** [review]

**Review by jrcreative — APPROVED** [review]
👍

**Comment by gravcmcguire** [comment]
This edit isn't needed.

**Pull request merged**